cmd/edgectl/aes_install.go

```diff
@@ -437,166 +429,100 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
-	// Allow overriding the source domain (e.g., for smoke tests before release)
-	manifestsDomain := "www.getambassador.io"
-	domainOverrideVar := "AES_MANIFESTS_DOMAIN"
-	overrideMessage := "Downloading manifests from override domain %q instead of default %q because the environment variable %s is set."
-	if amd := os.Getenv(domainOverrideVar); amd != "" {
-		i.ShowWrapped(fmt.Sprintf(overrideMessage, amd, manifestsDomain, domainOverrideVar))
-		manifestsDomain = amd
+	namespace, err := i.kubeinfo.Namespace()
+	if err != nil {
+		i.Report("fail_no_cluster")
+		return fmt.Errorf(noCluster)
 	}
 
-	// Download AES manifests
-	crdManifests, err := getManifest(fmt.Sprintf("https://%s/yaml/aes-crds.yaml", manifestsDomain))
+	// Try to determine cluster type from node labels
+	ci := i.GetClusterInfo()
+	i.SetMetadatum("Cluster Info", "cluster_info", ci.name)
+
+	i.SetMetadatum("Cluster Info", "managed", "helm")
```

Comment:
I think we only want to set the `managed` metadata if we run edgectl on top of an existing install, which is managed by `helm`. In the case of a first install, managed by edgectl, then this field should not be set.